manifests/source.pp

```diff
@@ -29,17 +28,9 @@
     $_release = $release
   }
 
-  # Some releases do not support https transport with default installation
```

Comment:
Removal of this is a regression. It's a feature we depend on.

Reply:
https://tickets.puppetlabs.com/browse/MODULES-7540 has been created, this was my fault. I have a fix incoming, and will start a release soon after.

Reply:
Thanks, just ran into it but hadn't had time to create an issue myself. Luckily for me I noticed it on a CI job and not in production.